liwords-ui/src/about/termsOfService.tsx

```diff
@@ -0,0 +1,444 @@
+            loans, nor do they imply any additional transfer of services. We
+            will refund donations if requested.
+          </p>
+          <h4>Stuff we’re not responsible for</h4>
+          <p>
+            Services made available are on an as-is and as-available basis.
+            Woogles makes no warranty or representation and disclaims all
+            responsibility and liability for the completeness, availability,
+            timeliness, accuracy and security of the content and services.
+          </p>
+          <p>
+            We disclaim all responsibility for any harm or loss of any data to
+            your computer system, the deletion of or failure to store or
+            transmit communications or content maintained by the services or
+            whether the services will meet your requirements or be available on
+            an uninterrupted, secure or error-free basis. Woogles cannot be held
+            responsible for any loss, financial or otherwise, from third party
+            sources, including misrepresentations, negligence or fraud.
+          </p>
+          <p>
+            No advice, information or written correspondence whether oral or
+            written procured from Woogles entities or through Woogles services
+            will create any warranty or representation not expressly made
+            herein.
+          </p>
+          <h4>Advertisement Policy</h4>
+          <p>
+            Woogles will never knowingly host advertisements. Woogles is not
+            responsible for third party advertisements, widgets and apps hosted
+            by service providers on Woogles social media.
+          </p>
+          <h4>Jurisdiction and Dispute Resolution</h4>
+          <p>
+            In the event of any dispute relating to these Terms or by using our
+            Services, we encourage you to contact our team first at woogles at
+            woogles.io such that we can attempt to work out an amicable
+            resolution.
+          </p>
+          <p>
+            You agree to accept the law and jurisdiction of the State of New
+            Jersey by using our services, without giving effect to any principle
+            of conflict of law. Any legal suit, action or proceeding arising out
+            of, or related to, these Terms of Service shall be filed only in the
+            state or federal courts located in Essex County, New Jersey, and you
+            hereby consent and submit to the venue and personal jurisdiction of
+            such courts for the purposes of such action.
+          </p>
+          <p>
+            In the event any provision of these terms is held to be invalid or
+            unenforceable, then that provision will be limited to the minimum
+            enforcement possible, and the remaining terms held with full effect.
+          </p>
+          <p>These terms are an agreement between you and Woogles.</p>
```

Comment:
"These terms are" (plural) and "an agreement" (singular) seem to disagree, not sure if it's correct or how to rephrase